docs/best-practices.md

```diff
@@ -240,6 +240,26 @@ include::example$installation/post-installation-steps.sh[]
 ----
 ```
 
+### OCC Examples
+
+When creating examples that show how to use occ, ensure that you use the `occ-command-example-prefix` attribute.
+Doing so will keep all examples of its use consistent throughout the documentation.
+
+**Note:** when used within a source code block, as in the following example, `subs="attributes"` has to be set, otherwise it won't render properly:
```

Comment:
`subs="attributes"` is incorrect, as the example below shows. It should be `subs="attributes+"`